src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -6,6 +6,21 @@
   = render partial: 'actions'
   = render partial: 'add_reviewer_dialog'
 
+.alert.alert-info{ role: 'alert' }
+  As part of the
+  = succeed ',' do
```

Comment:
This is needed to avoid `beta program ,`, so now we have `beta program,` as it should be.